contracts/StablecoinConverter.sol

```diff
@@ -109,7 +113,8 @@ contract StablecoinConverter is EpochTokenLocker {
     function addToken(address token) public {
         require(numTokens < MAX_TOKENS, "Max tokens reached");
         if (numTokens > 0) {
-            TokenOWL(tokenIdToAddressMap(0)).burnOWL(msg.sender, 10);
+            // Only charge fees for tokens other than the fee token itself
+            TokenOWL(tokenIdToAddressMap(0)).burnOWL(msg.sender, TOKEN_ADDITION_FEE_IN_OWL);
```

Comment:
(Not sure if I commented that before, but I cannot find the discussion)

Should we keep this as a TokenOWL instance variable on the contract instead of converting it on every addToken. At least the constructor should probably take a token of type `TokenOWL` to be more typesafe. 
Reading from a single variable could also be slightly cheaper than the BiMap lookup, but I'm not sure.

Reply:
> instead of converting it on every addToken

I would be happy to store the owl token address immediately in the constructor if this is what you are referring to!